wp-content/themes/pub/wporg-learn-2020/css/vendor/_sensei.scss

```diff
@@ -144,3 +144,6 @@ details {
 		line-height: 2.8;
 	}
 }
+.wp-block-sensei-lms-course-results.is-style-default .wp-block-sensei-lms-course-results__module-header:not(.has-background) {
+    background-color: #eee;
+}
```

Comment:
New lines above and below this please

```suggestion

.wp-block-sensei-lms-course-results.is-style-default .wp-block-sensei-lms-course-results__module-header:not(.has-background) {
    background-color: #eee;
}

```

Reply:
@adamwoodnz your screenshot doesn't show a module section divider in question here but rather just lesson links. So I'm wondering just like you if module names are somehow no longer available. I just accessed Sensei live to verify some courses I'm familiar with and I don't see data for modules.

Reply:
@jonathanbossenger @bsanevans are you still able to reproduce this please?

Reply:
Yup, accessing this page below still shows the same as seen in the screenshot of #1874 
https://learn.wordpress.org/course-complete/?course_id=41529

Reply:
Ok thanks @bsanevans, I still see this, but I'll trust you both 🙂 

![Screenshot 2024-01-08 at 3 15 34 PM](https://github.com/WordPress/Learn/assets/1017872/2926473a-74e4-49f8-a53e-96fdc9c5fdf1)